### ISSUE TITLE
Importing MultiMC instances with icons with the same names as built in icons overrides built in icons

### ISSUE THREAD

**Issue by kb-1000** [MultiMC/MultiMC5#3068]
System Information
-----------------------------
MultiMC version: 0.6.12-develop-1434

Operating System: Windows 10 1909

Summary of the issue or suggestion:
----------------------------------------------

What should happen:
------------------------------
The instance should get imported, with the icon in its ZIP, but not change the icons of other instances.

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. Create instance with MultiMC infinity icon

2. Import ScottoMotto's https://www.mediafire.com/file/mw8vl1txt7fkg1i/20w14Infinite_Fabric.zip/file

Suspected cause:
---------------------------

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)
![https://cdn.discordapp.com/attachments/134843027553255425/695226578912936026/unknown.png](https://cdn.discordapp.com/attachments/134843027553255425/695226578912936026/unknown.png)

Additional Info:
---------------------------

**Comment by Merith-TK** [comment]
Since my issue was closed as it was a duplicate

possible fix for this,
if `$INST_DIR` contains <iconKey>.`png`/`jpg`/`jpeg` display that icon, rather than updating the global index of icons (or whatever it is called instead)

**Comment by Zetabite** [comment]
From what i read in the code, overwriting the builtin ones is intended

**Comment by kb-1000** [comment]
I don't agree here... sure, it's probably intended that _you as the user_ can override them.
But do you really want that importing some random instance that someone with such a setup exported to overwrite the icon of _all_ existing instances that use it? I don't.
I had that happen with an instance replacing the `infinity` icon once.

**Comment by kb-1000** [comment]
reading up, that's actually what made me create this issue

**Comment by Zetabite** [comment]
well, with my suggested PR, there would most likely, no longer be overwritten icons, unless you actively rename the icon
